planner/core/testdata/analyze_suite_in.json

```diff
@@ -193,5 +193,12 @@
       // Should choose `primary` index instead of index `b`.
       "explain select c, b from t where a = 7639902 order by b asc limit 6"
     ]
+  },
+  {
+    "name": "TestLimitIndexEstimation",
+    "cases": [
+      // Should choose idx_a instead of idx_b, because idx_b would scan 990001 rows.
+      "explain select * from t where a <= 10000 order by b limit 1"
```

Comment:
Add some cases to choose index `idx_b` as a comparison？

Like `a >= 10000` `a <= 900000`?